File: Topic Based Problems/01 Fundamentals/2 Intermediate/01 Calculate BMI/C++/CalculateBMI.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    double weight, height;
    
    cout << "Enter your weight in kilogram: ";
    cin >> weight;

    cout << "Enter your height in meter: ";
    cin >> height;

    double bmi = weight / (height * height);

    cout << fixed << setprecision(2) << "Your BMI is " << bmi << endl;
    
    return 0;
}

File: Topic Based Problems/01 Fundamentals/2 Intermediate/01 Calculate BMI/Java/CalculateBMI.java
import java.util.Scanner;

public class CalculateBMI {
    public static void main(String[] args) {
        double weight, height;

        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter your weight in kilogram: ");
        weight = scanner.nextDouble();

        System.out.print("Enter your height in meter: ");
        height = scanner.nextDouble();

        double bmi = weight / (height * height);

        System.out.printf("Your BMI is %.2f%n", bmi);
        
        scanner.close();
    }
}
